lib/cocoapods/resolver.rb

```diff
@@ -164,7 +164,7 @@ def search_for(dependency)
 
         specifications_for_dependency(dependency, additional_requirements)
       end
-      @search[dependency].dup
+      @search[dependency]
```

Comment:
can remove this whole line now and save the hash lookup. worth freezing to ensure the array isnt unintentionally mutated?

Reply:
nice catch. I will test with `freeze` and see if anything comes up (& remove extra line)

Reply:
`freeze` does not appear to introduce any issues